dto/metrics_dto.go

```diff
@@ -0,0 +1,76 @@
+package dto
+
+import (
+	"time"
+
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/pure_utils"
+	"github.com/google/uuid"
+)
+
+type MetricCollectionFrequencyDto string
+
+const (
```

Comment:
To be discused, but I think we are not going to make any difference between metric frequency, they are all going to be treated the same, so those might be moot.